fix: Re-add missing symbolic icons

isantop opened the pull request:
It appears that some of our symbolic icons got removed at some point. This puts them back in place.

Fixes #63

jackpot51 commented:
@brs17 this may fix https://github.com/pop-os/icon-theme/issues/58

jackpot51 commented:
:musical_score: :musical_note: _I know this is right so I'm gonna merge it_ :musical_note: :musical_score:

Commit:
fix: Re-add missing symbolic icons

It appears that some of our symbolic icons got removed at some point. This puts them back in place.
